cogboard-webapp/src/components/AppDialog.js

```diff
@@ -43,7 +44,8 @@ AppDialog.propTypes = {
 }
 
 AppDialog.defaultProps = {
-  title: ''
+  title: '',
+  disableBackdropClick: false
```

Comment:
You don't have to set defaults, because it is MUI Dialog's component prop, so it is handled there.

Reply:
done